hack/build_artifacts.sh

```diff
@@ -0,0 +1,44 @@
+#! /bin/zsh
+
+setopt ERR_EXIT
```

Comment:
ok so I am taking it that this is an experiment to emulate `go test $(go list ./...|grep pkg)`, isn't it?

Reply:
yes, this is cruft.